Feature capstone project 1

kit0-0 opened the pull request:
**This pull request includes the following changes to the Capstone project:**

- [ ]  Add Home page (Mobile/Desktop).
- [ ]  Add About page (Mobile/Desktop).
- [ ] Added CSS for Home and About pages
- [ ] Implemented dynamic elements on the Home page ( speaker and Program section).

These changes greatly improve the overall user experience and functionality of the Capstone project.

shella12 commented on about.html:
```diff
@@ -0,0 +1,102 @@
+                <li><a href="#" class="button nav-list">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    </main>
```
- [ ] [OPTIONAL] Please remove the extra closing `</main>` tag from line 37 to as doesn't have any opening tag and seems to be placed by mistake.

shella12 commented on about.html:
```diff
@@ -0,0 +1,102 @@
+        </nav>
+    </header>
+    </main>
+    <main class="main-container about-container">
+        <div class="center-about">
+            <h1 class="top-h1-about">Hello! sharing world</h1>
+            <h2 class="main-sec-heading-about">CREATIVE CITY SUMMIT 2015</h1>
+            <p class="main-par-about">
+                Connecting with sector colleagues is an essential part of developing culture in Canada. Every year, delegates representing culture work across the country gather to share, learn, and grow..
+                Connecting with sector colleagues is an essential part of developing culture in Canada. Every year, delegates representing culture work across the country gather to share, learn, and grow..
+            </p>
+            <p class="contact-us">
+                Do you have other questions about the CC Global Summit 2015! </p>
+            <a href="#" class="email-id">ccsummit2015@cccanada.org </a>
+        </div>
+    </main>
```
- [ ] Please use `section` tag instead of `main` tag for your hero section of the app or you can inclose all your `section` tags inside the `main` tag. 

The` <main>` tag specifies the main content of a document. The content inside the `<main>` element should be unique to the document. It should not contain any content that is repeated across documents such as sidebars, navigation links, copyright information, site logos, and search forms. `<section>` element is used to represent a group of related content. You can use `<section>` inside the `<main>` tag.

shella12 commented on index.html:
```diff
@@ -0,0 +1,97 @@
+            </ul>
+        </nav>
+    </header>
+    <main class="main-container">
+        <h1 class="main-heading">Hello! sharing world</h1>
+        <h2 class="main-sec-heading">CREATIVE CITY SUMMIT 2015</h2>
+        <p class="main-par">Join us in Waterloo, 2015 for an exciting itinerary of speakers, peer
+            presentations, facilitated discussions, networking, and local culture offerings from the City of Waterloo
+            and regional partners. In-person and virtual tickets are on sale now!</p>
+        <p class="bold">
+            15 November 2015 FROM 08:00am
+        </p>
+        <p class="bottom-text">@ Canada , the City of Waterloo</p>
+    </main>
+
```
- [ ] Please use `section` tag instead of `main` tag for your hero section of the app or you can inclose all your `section` tags inside the `main` tag. 

The` <main>` tag specifies the main content of a document. The content inside the `<main>` element should be unique to the document. It should not contain any content that is repeated across documents such as sidebars, navigation links, copyright information, site logos, and search forms. `<section>` element is used to represent a group of related content. You can use `<section>` inside the `<main>` tag.

shella12 commented on style.css:
```diff
@@ -0,0 +1,851 @@
+  position: fixed;
+}
+
+.sec-navbar {
+  display: flex;
+  justify-content: space-between;
+}
```
- [ ] Please adjust the spacing between the `navbar `elements to match the design guidelines as according to the requirements we must follow the layout of the provided [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 

|Cindy shin's design|Your design|
|---|----|
|<img width="1501" alt="Screenshot 2023-05-29 at 12 29 19 PM" src="https://github.com/kit0-0/Capstone-summit/assets/44798044/7c16d36e-6b8b-45ea-840c-cb9f65350d79">|<img width="1501" alt="Screenshot 2023-05-29 at 12 24 47 PM" src="https://github.com/kit0-0/Capstone-summit/assets/44798044/3684dbc2-f8c1-4310-9af8-192fc2f995a9">|

shella12 reviewed (changes requested):
Hi @kit0-0, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media1.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Highlights

- Great design implementation ✅ 
- Clean code ✅   
- Good commit history ✅ 
- Descriptive pull request ✅ 
- Professional README ✅ 
- Deployed project using git pages ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

kit0-0 commented:
# New additions:

-Adjusted spacing between the logo and list items in the second navbar
-Replaced the `<main>` tag with the `<section>` tag for improved semantic structure

V-Blaze reviewed (approved):
Hi @kit0-0  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![giphy](https://user-images.githubusercontent.com/107632780/229644741-7dea3027-bf3f-4416-855b-46095e3bf6d5.gif)

## Highlights

- Descriptive pull request summary💯
-  Commit messages are descriptive👍
-  Great communication with the reviewer🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
